Assets/Scenes/TrainTaskV1/Scripts/Movable.cs

```diff
@@ -27,17 +24,14 @@ void Start()
     /// </summary>
     public void Update()
     {
-        if (lastPosition != transform.position || lastRotation != transform.eulerAngles)
-        {
-            DataCaptureSystem.Instance.ReportEvent(
-                $"{this.name}.transform.position",
-                transform.position
-            );
-            DataCaptureSystem.Instance.ReportEvent(
-                $"{this.name}.transform.eulerAngles",
-                transform.eulerAngles
-            );
-        }
+        DataCaptureSystem.Instance.ReportEvent(
+            $"{this.name}.transform.position",
+            transform.position
+        );
+        DataCaptureSystem.Instance.ReportEvent(
+            $"{this.name}.transform.eulerAngles",
+            transform.eulerAngles
+        );
```

Comment:
Should these all be columns? X Y Z ...

Reply:
What do you think? I think yes, but can also see an argument that this may be a lot of additional columns that spread out the information.

Reply:
Fixed.